src/main/java/io/mosip/mimoto/util/CredentialIssuerWellknownResponseValidator.java

```diff
@@ -17,5 +20,29 @@ public void validate(CredentialIssuerWellKnownResponse response, Validator valid
             }
             throw new InvalidWellknownResponseException(sb.toString());
         }
+
+        for (String name : response.getCredentialConfigurationsSupported().keySet()) {
+            //TODO: Extract the vc specific validations to separate classes
+            CredentialsSupportedResponse supportedCredentialConfiguration = response.getCredentialConfigurationsSupported().get(name);
+            if (supportedCredentialConfiguration.getFormat().equals("mso_mdoc")) {
+                if (supportedCredentialConfiguration.getDoctype().isBlank() || supportedCredentialConfiguration.getDoctype() == null) {
+                    throw new InvalidWellknownResponseException("Mandatory field 'doctype' missing");
+                }
+                if(supportedCredentialConfiguration.getClaims().isEmpty() || supportedCredentialConfiguration.getClaims()==null){
```

Comment:
Can we use StringUtils.isBlank method here as well

Reply:
claims is a map (Map<String, Object>) , so cant apply StringUtils function here

Reply:
we can use CollectionUtils as well in that case.